insonmnia/gatekeeper/gatekeeper.go

```diff
@@ -0,0 +1,351 @@
+package gatekeeper
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"math/big"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/noxiouz/zapctx/ctxlog"
+	"github.com/sonm-io/core/blockchain"
+	"github.com/sonm-io/core/util"
+	"go.uber.org/zap"
+)
+
+type Gatekeeper struct {
+	key *ecdsa.PrivateKey
+	cfg *Config
+
+	logger *zap.Logger
+
+	bch blockchain.API
+
+	in  blockchain.SimpleGatekeeperAPI
+	out blockchain.SimpleGatekeeperAPI
+
+	freezingTime time.Duration
+
+	spentToday *big.Int
+	lastDay    *time.Time
+
```

Comment:
what type of the ancient religion forced you to put white spaces after every couple lines in structure definition?